opennurbs_mesh.cpp

```diff
@@ -14831,7 +14831,9 @@ bool ON_MeshCache::Transform(
     ON_Mesh* mesh = item->m_mesh_sp.get();
     if (nullptr == mesh || mesh->IsEmpty())
       continue;
-    if (false == item->m_mesh_sp.unique())
+    // NOTE: use_count() == 1 is an approximation in multi-threaded environments
+    // https:// en.cppreference.com/w/cpp/memory/shared_ptr/unique
+    if (false == item->m_mesh_sp.use_count() == 1)
```

Comment:
This is really hard to read. I would recommend changing this to
`if (item->m_mesh_sp.use_count() != 1)`